modules/my_health/app/controllers/my_health/rx_controller.rb

```diff
@@ -12,7 +11,7 @@ class RxController < ApplicationController
     protected
 
     def client
-      @client ||= Rx::MedicationsClient.new(
+      @client ||= MyHealth::Prescriptions::Client.new(
```

Comment:
I don't understand the purpose of this change. Why are we even touching controllers in `my_health` at all?

We should be simply removing the legacy controllers and any classes that are no longer used because of it